GPyOpt/core/bo.py

```diff
@@ -228,29 +228,47 @@ def _compute_next_evaluations(self, pending_zipped_X=None, ignored_zipped_X=None
         ### We zip the value in case there are categorical variables
         return self.space.zip_inputs(self.evaluator.compute_batch(duplicate_manager=duplicate_manager, context_manager= self.acquisition.optimizer.context_manager))
 
+    def _normalize(self, Y, normalization_type='stats'):
+        """Normalize the vector Y."""
+        Y = np.asanyarray(Y)
+
+        # Only normalize with non null sdev (divide by zero). For only one
+        # data point both std and ptp return 0.
+        if normalization_type == 'stats':
+            Y_norm = Y - Y.mean()
+            std = Y.std()
+            if std > 0:
+                Y_norm /= std
+        elif normalization_type == 'maxmin':
+            Y_norm = Y - Y.min()
+            y_range = np.ptp(Y)
+            if y_range > 0:
+                Y_norm /= y_range
+                # [0, 1] is a strange range for a zero-mean GP
```

Comment:
It might be strange indeed, but it is now a behavior of the library. I would be very cautions changing it here. Let's separate things out: this PR is dealing with applying normalization to 1 point set, and let it be so. Changing the normalization range is a separate change, so if you still believe it is the right thing to do, can you please post a separate PR with just that so we can have a focused discussion?